telemetrylib/TCP.cpp

```diff
@@ -4,10 +4,13 @@
 //
 #include "DTI.h"
 
+
 class TCP : public DTI {
 public:
     void sendData(QByteArray bytes, long long timestamp) override {
         qDebug()<<"sending via TCP";
+        bytes.push_front("<bl>");
+        bytes.push_back("</bl>");
```

Comment:
```suggestion
        bytes.push_front("<bl>");
        bytes.push_back("</bl>");
```
Can you please add this to the SQL sending as well?